src/redux/walletconnect.ts

```diff
@@ -488,7 +484,7 @@ const listenOnNewMessages =
                   result: null,
                 });
                 const { accountAddress } = getState().settings;
-                logger.debug('[redux/walletconnect]: Updating session for chainID', { numericChainId }, logger.DebugContext.walletconnect);
+                logger.debug('WC: Updating session for chainID', { chainId: currentChainId }, logger.DebugContext.walletconnect);
```

Comment:
same here - this is using the `currentChainId` instead of the `chainId` from the new payload